webpack.config.js

```diff
@@ -9,6 +10,7 @@ const baseConfig = mode => ({
   module: {
     rules: [
       {
+        exclude: /node_modules/,
```

Comment:
I'm not sure about the implications here. Everything seems to work ok for me.

Reply:
Relevant: https://github.com/pmmmwh/react-refresh-webpack-plugin/issues/92#issuecomment-628960771